specification/operationalinsights/resource-manager/Microsoft.OperationalInsights/preview/2023-01-01-preview/SummaryLogs.json

```diff
@@ -0,0 +1,565 @@
+        },
+        "statusCode": {
+          "readOnly": true,
+          "type": "string",
+          "enum": [
+            "UserAction",
+            "DataPlaneError"
+          ],
+          "description": "An explicit reasoning stating the cause for the rule de activation.",
+          "x-ms-enum": {
+            "name": "StatusCodeEnum",
+            "modelAsString": true,
+            "values": [
+              {
+                "value": "UserAction",
+                "description": "Summary log rule stop originated from a user action (Stop was called)."
+              },
+              {
+                "value": "DataPlaneError",
+                "description": "Summary log rule stop was caused due to data plane related error."
+              }
+            ]
+          }
+        },
+        "provisioningState": {
+          "type": "string",
+          "readOnly": true,
+          "description": "SummaryLogs current provisioning state. If set to 'updating/deleting', indicates a resource lock due to ongoing operation, forbidding any update to the summary logs until the ongoing operation is concluded.",
+          "enum": [
+            "Updating",
+            "Succeeded",
+            "Deleting",
+            "Failed",
+            "Canceled"
+          ],
+          "x-ms-enum": {
+            "name": "ProvisioningStateEnum",
+            "modelAsString": true,
+            "values": [
+              {
+                "value": "Updating",
+                "description": "Summary logs is still being built and updated, summary logs is currently locked for any changes till the procedure is done."
+              },
+              {
+                "value": "Succeeded",
+                "description": "Summary logs state is stable and without changes, Summary logs is unlocked and open for new updates."
+              },
+              {
+                "value": "Deleting",
+                "description": "Summary logs state is deleting, summary logs is currently locked for any changes till the procedure is done."
+              },
+              {
+                "value": "Failed",
+                "description": "Summary logs state is stable and without changes."
+              },
+              {
+                "value": "Canceled",
+                "description": "Summary logs state is stable and without changes."
+              }
+            ]
+          }
+        },
+        "ruleDefinition": {
+          "$ref": "#/definitions/RuleDefinition",
+          "description": "Rule definition parameters."
+        }
+      },
+      "description": "SummaryLogs properties."
+    },
+    "SummaryLogs": {
+      "type": "object",
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/SummaryLogsProperties",
+          "description": "Summary Logs properties."
+        },
+        "systemData": {
```

Comment:
>systemData [](http://example.com/codeflow?start=9&length=10)

This is redundant as ProxyResource(v5) already has systemData

Reply:
Removed.